sdcm/tester.py

```diff
@@ -163,6 +163,21 @@ def __init__(self, methodName='test', name=None, params=None,
         self.scylla_dir = SCYLLA_DIR
         self.scylla_hints_dir = os.path.join(self.scylla_dir, "hints")
 
+    def get_duration(self, duration):
```

Comment:
The idea was to change the methods `run_stress_thread_bench`, `run_stress_thread`, `run_gemini`  to use the one function, which will get the duration. in Constructor, config parameter already read. So i think there is no need to reed it again in each method.
`self._duration = self.params.get(key='test_duration', default=60)`
For future, this will one place where need to configure the timeout, and we assign for future usage single point of test duration.

Reply:
Ok, let's do it in separate PR